src/components/App.js

```diff
@@ -5,14 +5,13 @@ import About from "./About";
 
 // pass this data down as props to the child component(s) that need it!
 import user from "../data/user";
-console.log(user);
 
 function App() {
   return (
     <div>
       <NavBar />
-      <Home />
-      <About />
+      <Home city = {user.city} name = {user.name} color = {user.color} />
+      <About bio = {user.bio} links ={user.links}/>
```

Comment:
Just like in HTML, there are no spaces next the `=`.
```suggestion
      <Home city={user.city} name={user.name} color={user.color} />
      <About bio={user.bio} links={user.links}/>
```